public/js/event-app.js

```diff
@@ -120,25 +120,34 @@ $(document).ready(function() {
         var firstScriptTag = document.getElementsByTagName('script')[0];
         firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
 
+        this.currentUser = new models.User(USER);
+        this.currentUserIsAdmin = function() {
+          return this.currentUser.isAdminOf(curEvent);
```

Comment:
The ``this`` value here isn't enclosed -- won't ``this.currentUser`` be undefined?